about.html

```diff
@@ -8,57 +8,92 @@
 
   <link href="https://fonts.googleapis.com/css?family=Roboto|Rubik" rel="stylesheet">
   <link rel="stylesheet" href="css/index.css">
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+<link href="https://fonts.googleapis.com/css2?family=Hind+Siliguri&display=swap" rel="stylesheet">
 
 </head>
 
 <body>
-    <div class="container about-page">
-     
-    About
-
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
-
+  <div class="container">
+   
+    <section class="top">
+      <header>
+
+          
+          <div class="Logo">
+              <img src="/img/lambda-black.png" alt="Logo"> 
+           </div>
+          
+           <div class="navbar">
+           <a href="/index.html">Home</a>
+           <a href="/about.html">About</a>
+           <a href="#">Products</a>
+           <a href="#">Blog</a >
+           <a href="#">Contact</a>
+           
+          </div>
+      </section>
+        </header>
+        <div class="banner">
+            <img src="/img/jumbo.jpg" alt="Banner">
+        </div>
+
+<div class="about2">
+    <h2>About</h2>
+      <p>Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
+</p>
+</div>
+   <section class="middle">
+<div class="text-container">
 
     <img src="img/about-plan.png" alt="strategy">
     
-    Strategy
+    <h2>Strategy</h2>
     
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
+    <p>Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
```

Comment:
awesome job adding semantic html, consider adding indentation and removing unnecessary spaces to help in readability